mandelbulber2/src/shaders.cpp

```diff
@@ -730,6 +730,7 @@ sRGBAfloat cRenderWorker::AmbientOcclusion(const sShaderInputData &input)
 		double opacity = 0.0;
 		double shadowTemp = 1.0;
 
```

Comment:
I want to slice these loop iterations into strides to exploit vectors.

![screenshot from 2016-09-29 16-45-10](https://cloud.githubusercontent.com/assets/7040493/18973572/2a76dfcc-866c-11e6-8dd6-29409c6155a7.png)
![screenshot from 2016-09-29 16-44-09](https://cloud.githubusercontent.com/assets/7040493/18973573/2a790ec8-866c-11e6-8b45-7b6b4332f30c.png)